components/package.json

```diff
@@ -22,8 +22,6 @@
   "main": "lib/index.js",
   "module": "node",
   "scripts": {
-    "test:coverage": "nyc --reporter=text mocha --reporter spec './{,!(node_modules)/**/}*.spec.js'",
-    "test": "mocha --require ts-node/register --reporter spec './{,!(node_modules)/**/}*.spec.ts'",
```

Comment:
Since we don't have tests for the components, removing these scripts allows us to further reduce the number of dependencies.